engines/cge/metaengine.cpp

```diff
@@ -200,96 +200,129 @@ Common::KeymapArray CGEMetaEngine::initKeymaps(const char *target) const {
 
 	Action *act;
 
+	act = new Action(kStandardActionLeftClick, _("Left click"));
+	act->setLeftClickEvent();
+	act->addDefaultInputMapping("MOUSE_LEFT");
+	act->addDefaultInputMapping("JOY_A");
+	keymap->addAction(act);
+
+	act = new Action(kStandardActionRightClick, _("Right click"));
+	act->setRightClickEvent();
+	act->addDefaultInputMapping("MOUSE_RIGHT");
+	act->addDefaultInputMapping("JOY_B");
+	keymap->addAction(act);
+
+	act = new Action(kStandardActionSkip, _("Exit/Skip"));
+	act->setCustomEngineActionEvent(kActionEscape);
+	act->addDefaultInputMapping("ESCAPE");
+	act->addDefaultInputMapping("JOY_X");
+	keymap->addAction(act);
+
+	act = new Action(kStandardActionSave, _("Save game"));
+	act->setCustomEngineActionEvent(kActionSave);
+	act->addDefaultInputMapping("F5");
+	act->addDefaultInputMapping("JOY_LEFT_SHOULDER");
+	keymap->addAction(act);
+
+	act = new Action(kStandardActionLoad, _("Load game"));
+	act->setCustomEngineActionEvent(kActionLoad);
+	act->addDefaultInputMapping("F7");
+	act->addDefaultInputMapping("JOY_RIGHT_SHOULDER");
+	keymap->addAction(act);
+
 	// I18N: 3-4 dialogs of game version info, (translation) credits, etc.
-	act = new Action("Game Info", _("Game Info"));
-	act->setKeyEvent(KEYCODE_F1);
+	act = new Action("INFO", _("Game Info"));
+	act->setCustomEngineActionEvent(kActionInfo);
 	act->addDefaultInputMapping("F1");
+	act->addDefaultInputMapping("JOY_LEFT_TRIGGER");
 	keymap->addAction(act);
 
 	// I18N: This opens a Quit Prompt where you have to choose
 	// [Confirm] or [Continue Playing] lines with Left Click.
-	act = new Action("Quit Prompt", _("Quit Prompt"));
-	act->setKeyEvent(KeyState(KEYCODE_x, 0, KBD_ALT));
+	act = new Action("QUIT", _("Quit Prompt"));
+	act->setCustomEngineActionEvent(kActionQuit);
 	act->addDefaultInputMapping("A+x");
+	act->addDefaultInputMapping("JOY_RIGHT_TRIGGER");
 	keymap->addAction(act);
 
 	// I18N: Here ALTered Item refers to the dice that has been altered/changed.
 	// In order to escape the dice game loop press Right/Left Alt
-	act = new Action("ALTered Item", _("ALTered Item"));
-	act->setKeyEvent(KEYCODE_LALT);
+	act = new Action("ALTITEM", _("ALTered Item"));
+	act->setCustomEngineActionEvent(kActionAltDice);
 	act->addDefaultInputMapping("LALT");
 	act->addDefaultInputMapping("RALT");
+	act->addDefaultInputMapping("JOY_CENTER");
```

Comment:
`JOY_CENTER` is only available on Android, so it would be better to use something more universal.

```suggestion
	act->addDefaultInputMapping("JOY_Y");
```